MekHQ/src/mekhq/campaign/personnel/enums/RandomDependentMethod.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2021 - The MegaMek Team. All Rights Reserved.
+ *
+ * This file is part of MekHQ.
+ *
+ * MekHQ is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MekHQ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MekHQ. If not, see <http://www.gnu.org/licenses/>.
+ */
+package mekhq.campaign.personnel.enums;
+
+import megamek.common.util.EncodeControl;
+
+import java.util.ResourceBundle;
+
+public enum RandomDependentMethod {
+    //region Enum Declarations
+    NONE("RandomDependentMethod.NONE.text", "RandomDependentMethod.NONE.toolTipText"),
+    AGAINST_THE_BOT("RandomDependentMethod.AGAINST_THE_BOT.text", "RandomDependentMethod.AGAINST_THE_BOT.toolTipText");
+    //endregion Enum Declarations
+
+    //region Variable Declarations
+    private final String name;
+    private final String toolTipText;
+    //endregion Variable Declarations
+
+    //region Constructors
+    RandomDependentMethod(final String name, final String toolTipText) {
+        final ResourceBundle resources = ResourceBundle.getBundle("mekhq.resources.Personnel", new EncodeControl());
+        this.name = resources.getString(name);
+        this.toolTipText = resources.getString(toolTipText);
+    }
+    //endregion Constructors
+
+    //region Getters
+    public String getToolTipText() {
+        return toolTipText;
+    }
+    //endregion Getters
+
+    //region Boolean Comparisons
+    public boolean isNone() {
+        return this == NONE;
+    }
+
+    public boolean isAtB() {
+        return this == AGAINST_THE_BOT;
+    }
+    //endregion Boolean Comparisons
+
+/*
+    public AbstractDependent getMethod(final CampaignOptions options) {
```

Comment:
Not used but want to keep it?

Reply:
When I actually build these into full modules, this will be uncommented.

It means the enums are in the exact same style as the Marriage/Divorce/Procreation ones already merged, so it'll be faster in the future.